functions/go/apply-setters/main.go

```diff
@@ -51,3 +55,23 @@ func getSetters(fc interface{}) (applysetters.ApplySetters, error) {
 	applysetters.Decode(rn, &fcd)
 	return fcd, nil
 }
+
+// resultsToItems converts the Search and Replace results to
+// equivalent items([]framework.Item)
+func resultsToItems(sr applysetters.ApplySetters) []framework.Item {
+	var items []framework.Item
+	if len(sr.Results) == 0 {
+		items = append(items, framework.Item{
+			Message: "no matches",
```

Comment:
I don't see you set `severity` anywhere.
Is `severity: info` the default value?

When there are no matches, should it be info, warning or error?

Reply:
Is severity: info the default value? Yes

I think info is good enough for no results.

Reply:
Updated.